src/providers/v4/subgraph-provider.ts

```diff
@@ -17,10 +17,26 @@ export interface V4SubgraphPool {
   tvlUSD: number;
 }
 
+export type V4RawSubgraphPool = {
+  id: string;
+  feeTier: string;
+  liquidity: string;
+  token0: {
+    symbol: string;
+    id: string;
+  };
+  token1: {
+    symbol: string;
+    id: string;
+  };
+  totalValueLockedUSD: string;
+  totalValueLockedETH: string;
+  totalValueLockedUSDUntracked: string;
+};
+
 const SUBGRAPH_URL_BY_CHAIN: { [chainId in ChainId]?: string } = {
-  [ChainId.SEPOLIA]:
-    '',
-}
+  [ChainId.SEPOLIA]: '',
```

Comment:
do you have this url?

Reply:
We do have a alchemy private subgraph, but we cannot reveal in the public repo, because of the API key secret. This used to be ok with the hosted service of our subgraphs URL, because hosted services are free. But hosted services for Uniswap got deprecated in June, so in v4, I think we don't have any hosted service to begin with.